gateway/src/apicast/policy/nginx_metrics/nginx_metrics.lua

```diff
@@ -73,6 +73,12 @@ local http_connections_metric =  prometheus('gauge', 'nginx_http_connections', '
 local shdict_capacity_metric = prometheus('gauge', 'openresty_shdict_capacity', 'OpenResty shared dictionary capacity', {'dict'})
 local shdict_free_space_metric = prometheus('gauge', 'openresty_shdict_free_space', 'OpenResty shared dictionary free space', {'dict'})
 
+local response_times = prometheus(
+  'histogram',
+  'response_time_seconds',
+  'Request response times in seconds'
```

Comment:
I think we can expand the description into something more human-readable like: "How many seconds it took to send a response to the client".

Reply:
Fixed :+1: